code/modules/power/supermatter/supermatter.dm

```diff
@@ -152,6 +169,11 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	if(is_main_engine && GLOB.main_supermatter_engine == src)
 		GLOB.main_supermatter_engine = null
 	QDEL_NULL(soundloop)
+	distort.icon = 'icons/effects/32x32.dmi'
+	distort.icon_state = "SM_remnant"
+	distort.pixel_x = 0
+	distort.pixel_y = 0
+	distort.forceMove(get_turf(src))
```

Comment:
what, why?

Add to /Destroy() a QDEL_NULL(distort) so the effect is deleted properly

Reply:
Oooh it's a remnant. Set `distort = null` then after moving it, so the reference is removed